client/tests/phpunit/phpunit.xml

```diff
@@ -10,7 +10,7 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
         stopOnError="true"
-        stopOnFailure="true"
+        stopOnFailure="false"
```

Comment:
Presumably this was just for testing?
```suggestion
        stopOnFailure="true"
```

Reply:
Ah yeah, I like to see all the broken tests on a change rather than stopping at the first. I'll revert.